CNV-69277: DOC: [UI] Add additional memory swap calculations to VM page (#105655)
openshift/openshift-docs: modules/virt-using-wasp-agent-to-configure-higher-vm-workload-density.adoc

* Web console
+
. In the {product-title} web console, go to *Virtualization* -> *Overview* -> *Settings* -> *General settings* -> *Memory density*.
. Set *Configure memory density* to on.
+

> The current memory density is now displayed.

Reviewer: is the `The current memory density is now displayed` text necessary? i'll leave it up to you

. Expand the *Current memory density* line.
. Choose the desired density by moving the *Requested memory density* slider. You can increase the density from 100% up to 400% in increments of 25%.
+

> This section also displays the actual and requested memory density across the cluster for comparison.

Reviewer 2: ```suggestion
The actual and requested memory densities across the cluster are shown for comparison.
```
